.editorconfig

```diff
@@ -0,0 +1,19 @@
+root = true
+
+[*]
+trim_trailing_whitespace = true
+end_of_line = lf
+charset = utf-8
+insert_final_newline = true
+
+[*.{js,css}]
+indent_style = space
+indent_size = 2
+max_line_length = 80
+
+[*.html]
+indent_style = space
+indent_size = 4
+
+[*.md]
+trim_trailing_whitespace = false
```

Comment:
Why are we not trimming trailing whitespaces here?

Reply:
Sir i ventured through some of the project to know the indentation uniformity among code in most of them i find `trim_trailing_whitespace = false`